setup.md

```diff
@@ -9,10 +9,10 @@ nav_order: 2
 # Set up GUAC with Docker Compose
 
 {: .note }
-If you’d prefer, you can set up [GUAC with Kubernetes](https://github.com/kusaridev/helm-charts/tree/main/charts/guac) instead.
+If you’d prefer, you can set up [GUAC with Kubernetes](https://github.com/kusaridev/helm-charts/tree/main/charts/guac) instead. These instructions are hosted in a third-party repo and not maintained by the GUAC team.
```

Comment:
Maybe instead of "not maintained by the GUAC team." be "may not be synchronized with the GUAC repo".